src/base/io/writeCbModel.m

```diff
@@ -1,51 +1,98 @@
-function outmodel = writeCbModel(model,format,fileName,compSymbolList,compNameList,sbmlLevel,sbmlVersion, solverParams)
 % Writes out COBRA models in various formats
+% The "writeCbModel" function relies on another function
+% "io/utilities/writeSBML.m" to convert a COBRA-Matlab structure into
```

Comment:
Something is not right here with the header of the function. This will cause problems when generating the documentation.